tasks/wfm_build.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+let build = require('../lib/build.js');
+let watch = require('node-watch');
+const taskName = 'wfmTemplate';
+
+module.exports = function(grunt) {
+
+  grunt.config.merge({
+    wfmTemplate: {
+      build: {},
+      watch: {}
+    }
+  });
+
+  grunt.registerMultiTask('wfmTemplate', 'A build tool for WFM modules', function() {
+    const taskConfig = grunt.config.get(this.name);
+    const moduleName = getAndCheckModuleName(this, grunt, taskConfig);
+    this.async();
+
+    if (this.target === 'build') {
+      build(moduleName);
+    } else if (this.target === 'watch') {
+      watch('./' + taskConfig.templateDir || 'lib/angular/template', () => build(templateDir));
```

Comment:
`build(templateDir)` it's not the right argument here. Grunt tasks are broken as generated templates would have path as name. Going to create separate PR to fix it.